storage/range.go

```diff
@@ -179,6 +181,7 @@ func NewRange(desc *proto.RangeDescriptor, rm RangeManager) (*Range, error) {
 	r := &Range{
 		Desc:        desc,
 		rm:          rm,
+		firstIndex:  raftInitialLogIndex + 1,
```

Comment:
It's not obvious to me why this would start at +1... Why not just at raftInitialLogIndex? Isn't that index present in the logs?